pages/index.vue

```diff
@@ -21,14 +21,9 @@ export default {
     SpluseinsCalendar,
     SpluseinsFooter
   },
-  data() {
-    return {
-    }
+  async fetch({ store }) {
+    await store.dispatch('splus/load');
```

Comment:
`pages` können eine asynchrone `fetch`-Methode haben (Komponente nicht), die nur auf dem Server aufgerufen wird und mit der man Daten in den store lädt. Hier sind das die API-Daten, damit macht der rendering-Server also einen request gegen das API-Backend. Das läuft beides auf dem selben Server, ist also effektiver, als wenn die Daten erst nach dem Client Rendering  nachgeladen werden müssen 😃